packages/cli/src/util/pruneOldFilesInDir.ts

```diff
@@ -0,0 +1,21 @@
+import fs from "node:fs";
+import path from "node:path";
+
+export function pruneOldFilesInDir(dirpath: string, maxAgeMs: number): void {
+  if (!fs.existsSync(dirpath)) {
+    return;
+  }
+
+  for (const entryName of fs.readdirSync(dirpath)) {
+    const entryPath = path.join(dirpath, entryName);
+
+    const stat = fs.statSync(entryPath);
+    if (stat.isDirectory()) {
+      pruneOldFilesInDir(entryPath, maxAgeMs);
+    } else if (stat.isFile()) {
+      if (Date.now() - stat.mtimeMs > maxAgeMs) {
```

Comment:
Changed from `cTimeMs` to `mtimeMs` as can't manipulate creating time from Nodejs which makes it hard to test.

I am not sure what is technically more correct. Currently, it should not matter for ssz objects because I would assume those are just written once and not modified.

For more general purpose, I would not consider a file which was just recently modified "old" so might be anyways better to use `mTimeMs`